### PR TITLE
Update to React Native 0.73.0

### PR DESCRIPTION
**Pull request by olcayertas**
Updated dependencies to:
- "@types/react-native": "^0.72.8"
- "prettier": "^3.1.1"
- "rollup": "^4.9.1"
- "rollup-plugin-typescript2": "^0.36.0"
- "typescript": "^5.3.3" Added "--bundleConfigAsCjs" parameter to build scripts.
- Disabled `objectHashIgnoreUnknownHack` option in rollup.config.js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,4 +64,4 @@ If you have suggestions for how this component could be improved, or want to rep
 For more, check out the [Contributing Guide](CONTRIBUTING.md).
 
 ## License
-[ISC](LICENSE) © 2020 Sebass van Boxel <hello@svboxel.com>
```
Comment (SvanBoxel) [inline README.md:67]: @olcayertas Why are you updating the npm and license source?

Comment (olcayertas) [reply README.md:67]: I was try to create the pull request for my own fork. I will close this one.

**Comment by olcayertas** [comment]
Created by mistake.

File: README.md (hunk 1 above)
Comment (SvanBoxel) [reply README.md:67]: Feel free to submit your changes updates upstream. We'll make sure to get you listed as a contributor.

**Pull request closed without merging**